subsys/bluetooth/controller/ll_sw/nordic/lll/lll_adv_pdu.h

```diff
@@ -138,6 +138,13 @@ static inline struct pdu_adv *lll_adv_sync_data_peek(struct lll_adv_sync *lll,
 
 	return (void *)lll->data.pdu[last];
 }
+
+#if defined(CONFIG_BT_CTLR_ADV_EXT_PDU_EXTRA_DATA_MEMORY)
+static inline void *lll_adv_sync_extra_data_peek(struct lll_adv_sync *lll)
+{
+	return lll->data.extra_data[lll->data.last];
+}
```

Comment:
It would be rather unusual in controller code to have static non-inline function in header file. Is it related with this: https://www.kernel.org/doc/html/v4.10/process/coding-style.html#the-inline-disease